cairo/ethereum/cancun/main.cairo

```diff
@@ -0,0 +1,63 @@
+%builtins output range_check bitwise keccak poseidon range_check96 add_mod mul_mod
+// In proof mode running with RustVM requires declaring all builtins and taking them as entrypoint
+// This is probably a mis-handling from the RustVM side.
```

Comment:
```suggestion
```

Reply:
it appears that i have a sever skill issue and when i said that I could not repro I was running the wrong program. I confirm that not having all builtins of the layout causes issues:

```
2025-03-14T12:36:47.119212Z ERROR cairo_run_program: vm::vm::runner: Failed to run program: No stop pointer found for builtin pedersen_builtin
thread '<unnamed>' panicked at crates/cairo-addons/src/vm/runner.rs:693:13:
Failed to run block, exiting
```